src/gfs_bufr.fd/meteorg.f

```diff
@@ -6,35 +6,36 @@ subroutine meteorg(npoint,rlat,rlon,istat,cstat,elevstn,
 
 !$$$  SUBPROGRAM DOCUMENTATION BLOCK
 !                .      .    .                                       .
-! SUBPROGRAM:    meteorg
+! SUBPROGRAM:    meteorg                           
```

Comment:
Can you clean out all of the extra whitespace that was added throughout this code? Thanks!

Reply:
done, please check the latest commit. @KateFriedman-NOAA @HuiyaChuang-NOAA

Reply:
Much better, just one last extra whitespace at the end of line 1108 in `src/gfs_bufr.fd/meteorg.f`. Thanks for cleaning those out, it makes it cleaner and easier to see the actual code changes now. :)

Reply:
@KateFriedman-NOAA done.